spec/controllers/external_users/claims_controller_spec.rb

```diff
@@ -829,7 +829,7 @@ def build_claim_in_state(state)
 
 def build_sortable_claims_sample(advocate)
   [:draft, :submitted, :allocated, :authorised, :rejected].each_with_index do |state, i|
-    Timecop.freeze(i.days.ago) do
+    travel(-i.days) do
```

Comment:
was there a reason for not using `travel_to(i.days.ago)` here?

Reply:
I had an idea that it might be a slight optimisation, as it doesn't create an instance of `Time` but it turns out that `travel` just calls `travel_to Time.now + duration` anyway. That being the case, `travel_to` makes more sense (and in all the other places too).